src/plugins/grpc/client.js

```diff
@@ -0,0 +1,100 @@
+'use strict'
+
+const { FORMAT_HTTP_HEADERS, Tags } = require('opentracing')
+
+function patch (grpc, tracer, config) {
+  if (config.client === false) return
+
+  config = config.client || config
+
+  function gRPCInterceptor (options, nextCall) {
+    const { path, responseStream } = options.method_definition
+    const scope = tracer.scopeManager().active()
+    const span = tracer.startSpan('grpc.client', {
+      childOf: scope && scope.span(),
+      tags: {
+        [Tags.SPAN_KIND]: Tags.SPAN_KIND_RPC_CLIENT,
+        'grpc.method_name': path,
+        'grpc.method_type': responseStream ? 'server_stream' : 'unary',
+        'resource.name': path.split('/').slice(-1),
+        'service.name': config.service || `${tracer._service}-grpc`
+      }
+    })
+
+    return new grpc.InterceptingCall(nextCall(options), {
+      sendMessage: (message, next) => {
+        if (config.arguments) {
+          const values = config.arguments(message)
+
+          for (const key in values) {
+            span.setTag(`grpc.arguments.${key}`, values[key])
```

Comment:
Looks like these are called `fields` according to the [gRPC guide](https://grpc.io/docs/guides/)

Reply:
Since these are part of the message, I would probably go with `grpc.message.fields.*`

Reply:
Ah, I struggled a bit with that as well but I found that `message` was a bit ambiguous as to whether it represents the request or the response message (in the case of the `client`, I'm only setting tags for the request message)

I ended up going with `arguments` since it's what `grpc` is calling internally to their variable. See [here](https://github.com/grpc/grpc-node/blob/38637b51038e3c0a7dae8339c7d5fbeab888adb4/packages/grpc-native-core/src/client.js#L30), [here](https://github.com/grpc/grpc-node/blob/309033f681d535c3ac31ce85ae6734f6bec7b470/packages/grpc-js/src/client.ts#L164) and [here](https://github.com/grpc/grpc-node/blob/309033f681d535c3ac31ce85ae6734f6bec7b470/packages/grpc-js/src/client.ts#L137).

I'm fine with changing to `grpc.message.fields.*` but I'd also have to change the callback in the config. What name would you suggest in that case? (`config.fields` for example?)